From: nabijaczleweli <nabijaczleweli@gmail.com>
Date: Thu, 30 Jan 2020 07:23:46 +0100
Subject: Fix typo in pir-8-as manpage

Found by evil-mode lintian:
  I: pir-8-emu-binutils: spelling-error-in-manpage usr/share/man/man1/pir-8-as.1.gz characer character
---
 man/pir-8-as.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/pir-8-as.md b/man/pir-8-as.md
index 7bc598e..8df9558 100644
--- a/man/pir-8-as.md
+++ b/man/pir-8-as.md
@@ -11,7 +11,7 @@ Assembler for the pir-8.
 
 Specified input files (or "-" for stdin) are assembled sequentially into the output file.
 
-The comment characer is ";".
+The comment character is ";".
 
 The assembly format is as follows:
 
